src/languages/taggerscript.js

```diff
@@ -35,7 +35,12 @@ export default function(hljs) {
 
   const ESCAPE_SEQUENCE = {
     className: 'symbol',
-    begin: /\\./
+    begin: /\\[^u]/
```

Comment:
This isn't needed, just change the order so that unicode comes first...

Reply:
Ok, I'll test and update